crates/bevy_pbr/src/atmosphere/aerial_view_lut.wgsl

```diff
@@ -21,9 +21,10 @@ fn main(@builtin(global_invocation_id) idx: vec3<u32>) {
     if any(idx.xy > settings.aerial_view_lut_size.xy) { return; }
 
     let uv = (vec2<f32>(idx.xy) + 0.5) / vec2<f32>(settings.aerial_view_lut_size.xy);
-    let ray_dir = uv_to_ray_direction(uv);
-    let r = view_radius();
-    let mu = ray_dir.y;
+    let ray_dir = uv_to_ray_direction(uv).xyz;
```

Comment:
why does this need an .xyz now?

Reply:
Good point. I need to look into why we return vec4 from this function in the first place

Reply:
That extra w was documenting a way to scale per-ray step length by the cosine to the camera forward to compensate perspective foreshortening at the screen edges. None of the call sites actually used w, and our raymarching computes in world-space distances already. The removed w wasn’t used anywhere - safe to discard.